app/src/main/java/com/keylesspalace/tusky/ComposeActivity.java

```diff
@@ -1123,7 +1161,14 @@ private void addMediaToQueue(@Nullable String id, QueuedMedia.Type type, Bitmap
         layoutParams.setMargins(margin, 0, margin, marginBottom);
         view.setLayoutParams(layoutParams);
         view.setScaleType(ImageView.ScaleType.CENTER_CROP);
-        view.setImageBitmap(preview);
+        if (preview != null) {
+            view.setImageBitmap(preview);
+        } else {
+            Picasso.with(this)
```

Comment:
on master we switched to Glide. 
```suggestion
            Glide.with(this)
```